share/spice/currency/currency.js

```diff
@@ -287,13 +287,15 @@
             }
 
             var endpoint = "/js/spice/currency/1/" + from + "/" + to;
-            $.get(endpoint, function(payload) {
-
-                // jsonp is returned from the API so we have to alter the contents
-                var response = JSON.parse(payload.trim().replace(/^[^\(]*\(/, '').replace(/\);$/, ''));
-                response = response.conversion;
-                Converter.resetConverter(response);
-
+            $.ajax({ 
+                url: endpoint,
+                dataType: 'text',
```

Comment:
Isn't this supposed to be `script`?

Reply:
Technically, yes but that's the problem I'm trying to patch here.

jQuery would execute the script as is:

`$.ajax({ url: 'https://sakurity.com/jqueryxss', dataType: 'script' });`

Digging a little deeper into this, as is in production this makes a needless call to the main `ddg_spice_currency` each time you use the dropdown to change the currency once the IA is loaded. We could just make this JSON.